Kotlin/sort/Shell Sort/src/ShellSort.kt

```diff
@@ -0,0 +1,59 @@
+// To Implement Shell Sort in Kotlin
+import java.util.*
+
+fun main(args:Array<String>){
+    println("Enter The number of elements in array:")
+    val sc = Scanner(System.`in`)
+    val n = sc.nextInt()
+    val ar = Array(n) { 0 }
+    println("Enter the elements of array:")
+    for (i in 0 until n){
+        ar[i] = sc.nextInt()
+    }
+    println("Sorted Array----->\n")
+    var i = n/2 //creating Gap
```

Comment:
Shift the comment above the code instead of putting it right beside the code.

Reply:
Done